notifications-yaml-reference_5be45bb02c7d3a31944dbc98.md

```diff
@@ -0,0 +1,207 @@
+
+ * [Overview](#overview)
+ * [Properties](#properties)
+ * [The rules property in more detail](#the-rules-property-in-more-detail)
+ * [See Also](#see-also)
+
+## Overview
+
+You can find out how to create new notifications by visiting the
+[Slack Notification](https://docs.semaphoreci.com/article/91-slack-notifications)
+and the [sem command line tool Reference](https://docs.semaphoreci.com/article/53-sem-reference)
+pages of the Semaphore 2.0 documentation.
+
+## Rules of Notifications
+
+Notifications are governed by certain rules:
+
+* Notifications can be sent only when a pipeline is finished. Notifications for
```

Comment:
Its a bit hard to follow the docs when the rules describe what is not possible instead of what is possible.

I propose the following:

- First describe what is possible in several bulletpoints
- Then, we can add a paragraph that exaplains what is not posssible. 

Also, I am not sure if we should concentrate too much on what is not possible, it highlights the negative parts of the softaware instead of the positive ones.

Reply:
Good poing.